ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitTip.kt

```diff
@@ -0,0 +1,164 @@
+package org.p2p.uikit.components
+
+import android.graphics.Point
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.PopupWindow
+import androidx.annotation.StringRes
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.constraintlayout.widget.ConstraintSet
+import androidx.core.view.updateLayoutParams
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetTipViewBinding
+import org.p2p.uikit.utils.toPx
+
+private enum class TipPosition {
+    TOP,
+    BOTTOM
+}
+
+/**
+ * Shows PopupWindow with arrow
+ * PopupWindow appears natively above or under the view
+ * Arrow is shown only on top or bottom of PopupWindow
+ * Arrow is pointing on the center of [anchorView]
+ */
+fun showTip(
+    anchorView: View,
```

Comment:
how's the idea make this as a extension to a achorView?

Reply:
Sounds good. I'll do it